config/config.go

```diff
@@ -54,6 +55,9 @@ func (a *App) Validate() error {
 				return fmt.Errorf("bot %s not defined", msgr)
 			}
 		}
+		if _, err := time.ParseDuration(msg.ExpireTime); err != nil {
+			return fmt.Errorf("Invalid expiretime %s", msg.ExpireTime)
```

Comment:
best `err` ook loggen of komt daar niet veel nuttigs uit?

Reply:
Basicly says exactly the same but instead of expiretime it talks about duration
`time: invalid duration ergre20s   ` <- example